.gitignore

```diff
@@ -4,4 +4,5 @@ node_modules
 .env.local
 .env.development.local
 .env.test.local
-.env.production.local
\ No newline at end of file
+.env.production.local
+.notes.js
```

Comment:
To add the notes.js to gitignore you have to remove the `.` before the file name.